app/perfunwind.cpp

```diff
@@ -653,6 +656,12 @@ void PerfUnwind::analyze(const PerfRecordSample &sample)
         }
     }
 
+    if (qEnvironmentVariableIsSet("STITCH_CALLCHAIN")) {
```

Comment:
again, see above in loc 626, `frames.append` is called there, also update the ipStack there then

Reply:
also, this should become a command line switch instead of relying on an environment variable. i.e. mark this as FIXME and only keep the env var for testing for now

Reply:
furthermore, all the inserts and updates of the ipCache should not be done when this feature is not enabled

Reply:
done